src/main/java/com/example/animalshelter/controller/AnimalController.java

```diff
@@ -4,19 +4,24 @@
 import com.example.animalshelter.service.AnimalService;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-@RestController()
+@RestController
+@RequestMapping("/animals")
 public class AnimalController {
 
+    private static final String SUCCESSFUL_RESPONSE = "Animal added successfully.";
```

Comment:
Proposition: The animal has been added successfully.

Reply:
Done.